### PR TITLE
fix: reduce embed weight

### PR DESCRIPTION
**Pull request by tataihono**
# Description

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 30033c6</samp>

Refactored the `DiscoveryJourneys` and `EmbedJourney` components to remove the dependency on GraphQL data and simplify the testing and storybook setup. Deleted the unused `data.ts` file and cleaned up the imports and props in the related files.

Incorporated are a number of playwright fixes and optimizations as I had to work through some failures in the journeys screenshots.

### CODE REVIEW

File: apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx [hunk 1]
```diff
@@ -87,22 +85,19 @@ export function EmbedJourney({
               >
                 <Box
                   sx={{
-                    height: '100%',
-                    width: '100%',
-                    borderRadius: 4,
```
Comment (csiyang) [inline apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx:92]: add this back

File: apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx [hunk 2]
```diff
@@ -87,22 +85,19 @@ export function EmbedJourney({
...
-                    position: 'relative',
-                    cursor: 'pointer'
+                    display: 'flex',
+                    justifyContent: 'center',
+                    flexDirection: 'column',
+                    height: '100%'
                   }}
                 >
-                  <BlockRenderer block={block} />
-                  <StepFooter title={discoveryJourney?.seoTitle ?? ''} />
```
Comment (csiyang) [inline apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx:99]: No more stepFooter? It seems like they were using the host name?

File: apps/journeys-admin/src/components/DiscoveryJourneys/DiscoveryJourneys.spec.tsx [hunk 3]
```diff
@@ -1,15 +1,10 @@
...
 import { DiscoveryJourneys } from '.'
 
 describe('DiscoveryJourneys', () => {
   it('should show embed journeys', () => {
-    const { getByLabelText } = render(
-      <MockedProvider>
-        <DiscoveryJourneys />
-      </MockedProvider>
-    )
+    const { getByLabelText } = render(<DiscoveryJourneys />)
```
Comment (csiyang) [inline apps/journeys-admin/src/components/DiscoveryJourneys/DiscoveryJourneys.spec.tsx:7]: Just to cover the warning from `DiscoverJourneys.tsx`?

```suggestion
    const { getByLabelText } = render(<DiscoveryJourneys />)
    expect(getByTestId('JourneysAdminDiscoveryJourneys').toBeInTheDocument()
```

**Review by csiyang — COMMENTED** [review]
- Do we need ti get this look exactly the same as the current ones? If so, some of the spacing within the card is wrong. (Button)
-  run translations

File: apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx (hunk 2 above)
Comment (tataihono) [reply apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx:99]: not needed.

File: apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx (hunk 1 above)
Comment (tataihono) [reply apps/journeys-admin/src/components/DiscoveryJourneys/EmbedJourney/EmbedJourney.tsx:92]: fixed this.

**Review by csiyang — APPROVED** [review]

File: apps/journeys-e2e/src/e2e/journeys.spec.ts
Comment (tataihono) [inline apps/journeys-e2e/src/e2e/journeys.spec.ts:1]: @kiran-redhat got rid of the video playback in this PR. This was causing unexpected failures preventing us from shipping. We should reintroduce this behaviour once we have a test journey built out.

**Review by csiyang — APPROVED** [review]

**Review by mikeallisonJS — APPROVED** [review]

**Pull request merged**